core/tests/test_utils.py

```diff
@@ -249,7 +249,7 @@ class TestBase(unittest.TestCase):
         }
     }
 
-    VERSION_27_STATE_DICT = {
+    VERSION_31_STATE_DICT = {
```

Comment:
This should not be changed. Why are you changing it?

Ditto for a bunch of "27" cases below. The whole point of some of the existing tests is to ensure that entities are upgraded smoothly on-the-fly.